cpk_lib_python_github/github_app_token_generator_package/github_app_token_generator/main.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""GitHub App Token Generator - Main entry point."""
+import logging
+import sys
+
+from colorama import init
+
+from .cli import create_parser, print_banner
+from .config import get_config_from_env
+from .formatters import OutputFormatter
+from .github_api import GitHubAPIClient
+from .token_manager import TokenManager
+
+# Initialize colorama
+init(autoreset=True)
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    handlers=[logging.FileHandler("github_app_token.log"), logging.StreamHandler()],
+)
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    """Main entry point."""
+    print_banner()
+
+    parser = create_parser()
+    args = parser.parse_args()
+
+    # Set debug logging if requested
+    if args.debug:
+        logging.getLogger().setLevel(logging.DEBUG)
+        logger.debug("Debug logging enabled")
```

Comment:
```suggestion
    if args.debug:
        logging.getLogger().setLevel(logging.DEBUG)
        logger.setLevel(logging.DEBUG)
        logger.debug("Debug logging enabled")
```

Currently on `DEBUG` logs will be output

Reply:
Addressed